core_debug.tex

```diff
@@ -46,6 +46,11 @@ \section{Debug Mode} \label{debugmode}
 \item Effective XLEN is DXLEN.
 \end{steps}
 
+While in Debug Mode, the hart is waiting for commands from the external
```

Comment:
Thoughts?

Reply:
This whole list is confusing, because it mentions some things that are always true in Debug Mode (Effective XLEN is DXLEN.) and some things that only make sense when executed from the Program Buffer (All control transfer instructions...). I'm going to go through that and see if I can separate it out. Then probably your change will fit in one of the two (or both) lists.

Reply:
Moving things around a bit, can you look at #496?